src/robotto.js

```diff
@@ -82,4 +82,44 @@ robotto.parse = function(robotsFile) {
     return rulesObj;
 };
 
+robotto._stripRoute = function(urlParam) {
+    // Removes domain name and returns only the endPoint
+    return urlParam.replace(/^.*\/\/[^\/]+/, '');
+};
+
+robotto.check = function(userAgent, urlParam, rulesObj) {
+    let userAgents = Object.keys(rulesObj);
+    let desiredRoute = this._stripRoute(urlParam) + '/';
+    let allowed = true;
+
+    // Searches for every user agent until it gets a match
+    userAgents.forEach((agent) => {
+        if (agent.indexOf(userAgent) === 0) {
```

Comment:
Well, then we need to do something to partially match `/browse/` but not `/brow` :D
We need to partially match urls, but fully match between slashes, do you get it?